x-pack/plugins/security_solution/public/detections/pages/alerts/alert_details_redirect.test.tsx

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Router } from 'react-router-dom';
+import { AlertDetailsRedirect } from './alert_details_redirect';
```

Comment:
Not directly, related to this PR. But should path `/alerts/redirect` redirect to `alerts` if `alertId` is not provided.

Reply:
Well this is actually accurate behavior. It doesn't actually match a path since `:/alertId` is not set